### ISSUE TITLE
change travis-ci for more Python versions

### ISSUE THREAD

**Issue by prjemian** [prjemian/punx#98]
similar to how [spec2nexus project](https://github.com/prjemian/spec2nexus/blob/master/.travis.yml) has it setup now

**Comment by prjemian** [comment]
wrestling with this:

```
UnsatisfiableError: The following specifications were found to be in conflict:

  - pyqt 4* -> python 2.6* -> openssl 1.0.1*

  - python 3.6*

Use "conda info <package>" to see the dependencies for each package.

The command "conda install  pyqt=4 openssl=1.0.2l" failed and exited with 1 during .
```

**Comment by prjemian** [comment]
Can't build with PyQt=4 and Python 3.6

```

**Comment by prjemian** [comment]
Only PyQt=5 is available for Python 3.6

pyqt 5.6.0 py36_2
-----------------
file name   : pyqt-5.6.0-py36_2.tar.bz2
name        : pyqt
version     : 5.6.0
build string: py36_2
build number: 2
channel     : defaults
size        : 5.4 MB
arch        : x86_64
date        : 2017-01-17
license     : Commercial, GPLv2, GPLv3
license_family: GPL3
md5         : 379323f4abf4cced0864fb6d126b88cb
noarch      : None
platform    : linux
url         : https://repo.continuum.io/pkgs/free/linux-64/pyqt-5.6.0-py36_2.tar.bz2
dependencies:
    libgcc
    python 3.6*
    qt 5.6.*
    sip >=4.18
```
